### ISSUE TITLE
Strange inpaint artifacts

### ISSUE THREAD

**Issue by AlexYez** [lllyasviel/Fooocus#966]
Hi, I'm getting strange results when inpainting some images, mostly from the Dalle 3 neural network, what could be the issue? Outpaint works fine with this picture...

![scrnli_11_16_2023_5-31-45 PM](https://github.com/lllyasviel/Fooocus/assets/83212752/026efdc2-2dfc-4e23-9572-959fdd8baf9c)

**Full Console Log**
X:\Fooocus>.\python_embeded\python.exe -s Fooocus\entry_with_update.py --theme dark
Already up-to-date
Update succeeded.
[System ARGV] ['Fooocus\\entry_with_update.py', '--theme', 'dark']
Python 3.10.9 (tags/v3.10.9:1dd9be6, Dec  6 2022, 20:01:21) [MSC v.1934 64 bit (AMD64)]
Fooocus version: 2.1.820
Running on local URL:  http://127.0.0.1:7865

To create a public link, set `share=True` in `launch()`.
Total VRAM 8192 MB, total RAM 65237 MB
xformers version: 0.0.20
Set vram state to: NORMAL_VRAM
Disabling smart memory management
Device: cuda:0 NVIDIA GeForce RTX 3070 Laptop GPU : native
VAE dtype: torch.bfloat16
Using xformers cross attention
Refiner unloaded.
model_type EPS
adm 2816
Using xformers attention in VAE
Working with z of shape (1, 4, 32, 32) = 4096 dimensions.
Using xformers attention in VAE
extra keys {'cond_stage_model.clip_l.text_projection', 'cond_stage_model.clip_g.transformer.text_model.embeddings.position_ids', 'cond_stage_model.clip_l.logit_scale'}
Base model loaded: D:\Apps\Fooocus\xdrive\Fooocus\Fooocus\models\checkpoints\juggernautXL_version6Rundiffusion.safetensors
Request to load LoRAs [['sd_xl_offset_example-lora_1.0.safetensors', 0.1], ['None', 1.0], ['None', 1.0], ['None', 1.0], ['None', 1.0]] for model [D:\Apps\Fooocus\xdrive\Fooocus\Fooocus\models\checkpoints\juggernautXL_version6Rundiffusion.safetensors].
Loaded LoRA [X:\Fooocus\Fooocus\models\loras\sd_xl_offset_example-lora_1.0.safetensors] for model [D:\Apps\Fooocus\xdrive\Fooocus\Fooocus\models\checkpoints\juggernautXL_version6Rundiffusion.safetensors] with 788 keys at weight 0.1.
Fooocus V2 Expansion: Vocab with 642 words.
Fooocus Expansion engine loaded for cuda:0, use_fp16 = True.
Requested to load SDXLClipModel
Requested to load GPT2LMHeadModel
Loading 2 new models
[Fooocus Model Management] Moving model(s) has taken 0.54 seconds
App started successful. Use the app with http://127.0.0.1:7865/ or 127.0.0.1:7865
[Parameters] Adaptive CFG = 7
[Parameters] Sharpness = 2
[Parameters] ADM Scale = 1.5 : 0.8 : 0.3
[Parameters] CFG = 4.0
[Parameters] Seed = 183735578495833484
[Fooocus] Downloading inpainter ...
[Inpaint] Current inpaint model is X:\Fooocus\Fooocus\models\inpaint\inpaint_v26.fooocus.patch
[Parameters] Sampler = dpmpp_2m_sde_gpu - karras
[Parameters] Steps = 30 - 15
[Fooocus] Initializing ...
[Fooocus] Loading models ...
Refiner unloaded.
Request to load LoRAs [('sd_xl_offset_example-lora_1.0.safetensors', 0.1), ('None', 1), ('None', 1), ('None', 1), ('None', 1), ('X:\\Fooocus\\Fooocus\\models\\inpaint\\inpaint_v26.fooocus.patch', 1.0)] for model [D:\Apps\Fooocus\xdrive\Fooocus\Fooocus\models\checkpoints\juggernautXL_version6Rundiffusion.safetensors].
Loaded LoRA [X:\Fooocus\Fooocus\models\loras\sd_xl_offset_example-lora_1.0.safetensors] for model [D:\Apps\Fooocus\xdrive\Fooocus\Fooocus\models\checkpoints\juggernautXL_version6Rundiffusion.safetensors] with 788 keys at weight 0.1.
Loaded LoRA [X:\Fooocus\Fooocus\models\inpaint\inpaint_v26.fooocus.patch] for model [D:\Apps\Fooocus\xdrive\Fooocus\Fooocus\models\checkpoints\juggernautXL_version6Rundiffusion.safetensors] with 960 keys at weight 1.0.
Requested to load SDXLClipModel
Loading 1 new model
unload clone 1
[Fooocus Model Management] Moving model(s) has taken 1.12 seconds
[Fooocus] Processing prompts ...
[Fooocus] Preparing Fooocus text #1 ...
[Prompt Expansion] cyborg hand, intricate, futuristic, elegant, highly detailed, cinematic, extremely, sharp focus, fine detail, glowing, deep colors, epic composition, pristine, beautiful, symmetry, vivid, coherent, color, illuminated, striking, very inspirational, colorful, background, ambient light, professional, winning, creative, perfect, positive, unique, pure, full, atmosphere
[Fooocus] Encoding positive #1 ...
[Fooocus] Encoding negative #1 ...
[Fooocus] Image processing ...
[Fooocus] VAE Inpaint encoding ...
[Fooocus] VAE encoding ...
Final resolution is (1024, 1638), latent is (1008, 1008).
Preparation time: 8.52 seconds
[Sampler] refiner_swap_method = joint
[Sampler] sigma_min = 0.02916753850877285, sigma_max = 14.614643096923828
Requested to load SDXL
Loading 1 new model
[Fooocus Model Management] Moving model(s) has taken 3.37 seconds
100%|██████████████████████████████████████████████████████████████████████████████████| 30/30 [00:18<00:00,  1.61it/s]
Image generated with private log at: D:\Apps\Fooocus\xdrive\Fooocus\Fooocus\outputs\2023-11-16\log.html
Generating and saving time: 23.76 seconds
Requested to load SDXLClipModel
Requested to load GPT2LMHeadModel
Loading 2 new models
[Fooocus Model Management] Moving model(s) has taken 0.58 seconds
Total time: 33.49 seconds

Source image: 
![image-example](https://github.com/lllyasviel/Fooocus/assets/83212752/7f6a233c-77c6-499e-8542-a6fbf505c6bd)

**Comment by AlexYez** [comment]
In version 2.1.822 this has been fixed! Thank you for this amazing tool!
